test-cases/longevity/longevity-1TB-3days-authorization-and-tls-ssl.yaml

```diff
@@ -0,0 +1,41 @@
+test_duration: 5020
+prepare_write_cmd: ["cassandra-stress write cl=QUORUM n=275050075 -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native -rate threads=150 -col 'size=FIXED(200) n=FIXED(5)' -pop seq=1..275050075",
```

Comment:
Why is it "compaction(strategy=LeveledCompactionStrategy)"?
The whole idea behind this was to test it with STCS (as your pipeline name suggests...)

Reply:
Sorry, I did this mess when copied from the original file. Fixed